docs/source/en/using-diffusers/lcm.md

```diff
@@ -0,0 +1,192 @@
+
+Let's first initialize the `StableDiffusionXLControlNetPipeline` with the above `unet` and the `LCMScheduler`:
+
+```python
+import cv2
+import numpy as np
+import torch
+from PIL import Image
+
+from diffusers.utils import load_image
+from diffusers import (AutoencoderKL, ControlNetModel, LCMScheduler,
+                       StableDiffusionXLControlNetPipeline,
+                       UNet2DConditionModel)
+
+unet = UNet2DConditionModel.from_pretrained(
+    "lcm-sd/lcm-sdxl-full-param-1k-huber-simian",
+    torch_dtype=torch.float16,
+    variant="fp16",
+)
+controlnet = ControlNetModel.from_pretrained(
+    "diffusers/controlnet-canny-sdxl-1.0", torch_dtype=torch.float16
+)
+vae = AutoencoderKL.from_pretrained(
+    "madebyollin/sdxl-vae-fp16-fix", torch_dtype=torch.float16
+)
+
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+pipe = StableDiffusionXLControlNetPipeline.from_pretrained(
+    model_id, unet=unet, controlnet=controlnet, vae=vae, torch_dtype=torch.float16
+).to("cuda")
+pipe.scheduler = LCMScheduler.from_config(pipe.scheduler.config)
+```
+
+Prepare the conditioning image for inference:
+
+```python
+image = load_image(
+    "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/sd_controlnet/hf-logo.png"
+)
+image = np.array(image)
+image = cv2.Canny(image, 100, 200)
+image = image[:, :, None]
+image = np.concatenate([image, image, image], axis=2)
+image = Image.fromarray(image)
+```
+
+Perform inference:
+
+```python
+prompt = (
+    "aerial view, a futuristic research complex in a bright foggy jungle, hard lighting"
+)
+image = pipe(
+    prompt=prompt,
+    image=image,
+    num_inference_steps=4,
+    guidance_scale=1,
+    controlnet_conditioning_scale=0.5,  # recommended for good generalization
+).images[0]
+```
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/lcm/lcm_controlnet.png)
+
+## LoRA
+
+It is possible to generalize the LCM framework to use with [LoRA](../training/lora.md). It effectively eliminates the need to conduct expensive fine-tuning runs as LoRA training concerns just a few number of parameters compared to full fine-tuning. During inference, the [`LCMScheduler`] comes to the advantage as it enables very few-steps inference without compromising the quality.
+
+```python
+from diffusers import DiffusionPipeline, LCMScheduler
+import torch
+
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+lcm_lora_id = "lcm-sd/lcm-sdxl-lora-huber-final"
+
+pipe = DiffusionPipeline.from_pretrained(model_id, variant="fp16", torch_dtype=torch.float16).to("cuda")
+
+pipe.load_lora_weights(lcm_lora_id, weight_name="lcm_sdxl_lora.safetensors")
+pipe.scheduler = LCMScheduler.from_config(pipe.scheduler.config)
+
+prompt = "close-up photography of old man standing in the rain at night, in a street lit by lamps, leica 35mm summilux"
+image = pipe(
+    prompt=prompt,
+    num_inference_steps=4,
+    guidance_scale=1,
```

Comment:
```suggestion
    guidance_scale=0,
```

Actually `guidance_scale=0` also disables it and is cleaner.

Reply:
@pcuenca tested it with 1.0.

Reply:
0 and 1 is the same